dev/io.openliberty.cdi.3.0.internal_fat/fat/src/io/openliberty/cdi30/internal/fat/tests/CDI30BeansXMLTest.java

```diff
@@ -52,7 +52,7 @@ public static void setUp() throws Exception {
                                               .addClass(CDI30BeansXMLTestServlet.class.getName())
                                               .addClass(SimpleBean.class.getName())
                                               .addClass(AnnotatedBean.class.getName());
-        cdiBeansXMLWar = CDIArchiveHelper.addBeansXML(cdiBeansXMLWar, Mode.ALL, Version.CDI30);
+        cdiBeansXMLWar = CDIArchiveHelper.addBeansXML(cdiBeansXMLWar, DiscoveryMode.ALL, CDIVersion.CDI30);
```

Comment:
Given that all the helper methods modify the argument anyway, can please not do the assignment on this line?

The only reason to that all the shrinkwrap methods return `this` is so that you can chain method calls together for brevity and given that we're not doing that, I don't think it makes sense to pretend that you might need to assign the returned value.

Reply:
ok, don't much like method chaining anyway